src/Functions/generateSerialID.cpp

```diff
@@ -88,14 +90,19 @@ class FunctionSerial : public IFunction
     bool useDefaultImplementationForNothing() const override { return false; }
     bool canBeExecutedOnDefaultArguments() const override { return false; }
     bool isSuitableForShortCircuitArgumentsExecution(const DataTypesWithConstInfo &) const override { return false; }
-    ColumnNumbers getArgumentsThatAreAlwaysConstant() const override { return {0}; }
+    ColumnNumbers getArgumentsThatAreAlwaysConstant() const override { return {0, 1}; }
 
     DataTypePtr getReturnTypeImpl(const ColumnsWithTypeAndName & arguments) const override
     {
         FunctionArgumentDescriptors mandatory_args{
             {"series_identifier", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isString), &isColumnConst, "const String"}
         };
-        validateFunctionArguments(*this, arguments, mandatory_args);
+
+        FunctionArgumentDescriptors optional_args{
+            {"start_value", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isUInt), &isColumnConst, "const UInt"}
```

Comment:
When the argument is not bounded to a specific size of integer, we indicate that with a `*`.
```suggestion
            {"start_value", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isUInt), &isColumnConst, "const UInt*"}
```